run.sh

```diff
@@ -255,15 +275,22 @@ check_themes() {
     done
   fi
 
-  [[ "${#to_install}" -gt 0 ]] && wp theme install --allow-root "${to_install[@]}" | tail -n 1 |& _colorize
+  for key in "${to_install[@]}"; do
+    wp theme install --allow-root "$key" 
+    _log_last_exit_colorize "Success: $key theme install " "Error: $key theme install failed!"
+  done 
 
   for theme in $(wp theme list --field=name --status=inactive --allow-root); do
     [[ ${theme_deps[$theme]} ]] && continue
     [[ ${theme_volumes[$theme]} ]] && continue
     to_remove+=( "$theme" )
   done
 
-  [[ "${#to_remove}" -gt 0 ]] && _wp theme delete "${to_remove[@]}"
+  for key in "${to_remove[@]}"; do
```

Comment:
Same as above

Reply:
wp theme intall does the same thing.  It inherits from the same [base class](https://github.com/wp-cli/extension-command/blob/master/src/Theme_Command.php#L40) and calls the[ parent class install](https://github.com/wp-cli/extension-command/blob/master/src/Theme_Command.php#L508) the same way